dev_scripts/find_images_to_rebuild.py

```diff
@@ -50,5 +62,21 @@
     dry_run=True, **kwargs)
 
 handler = RebuildImagesOnRPMAdvisoryChange()
+
+if args.cassete_path:
+    VCRPY_MODE = "none"
+    try:
+        file_dict = yaml.safe_load(args.cassete_path)
+        response_container_repo = json.loads(
+            file_dict['interactions'][0]['response']['body']['string'])
+        response_container_imgs = json.loads(
+            file_dict['interactions'][1]['response']['body']['string'])
+        with patch("freshmaker.consumer.get_global_consumer"), patch(
```

Comment:
vcrpy should handle all this for you. You shouldn't need to do any mocking.

Reply:
Oh. didn't know it. Now it looks much better. PTAL now. Did I understood right how it should be.